app/src/main/res/layout/recyclerview_row.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content">
+    <LinearLayout
```

Comment:
You're not really using the outer constraint layout so it can be deleted. Then the current line 5 can be replaced with:
`<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"`